chef/data_bags/crowbar/bc-template-nova_dashboard.schema

```diff
@@ -40,6 +40,7 @@
             },
             "can_set_mount_point": { "type": "bool", "required": false },
             "can_set_password": { "type": "bool", "required": false },
+            "multidomain": { "type": "bool", "required": false },
```

Comment:
since the migration below creates it, imho we should set this to required true..